packages/integration-testsuite/src/apolloServerTests.ts

```diff
@@ -2777,24 +2776,6 @@ export function defineIntegrationTestSuiteApolloServerTests(
         );
       });
 
-      // We can remove this test after the initial AS4 release once we stop supporting
-      // the playground package.
-      it('can install playground with specific version', async () => {
```

Comment:
We should probably do this on main afterwards?

Reply:
https://github.com/apollographql/apollo-server/pull/7586